contentful_migrations/migrations/1661197009323--new-compose--add-fields.cjs

```diff
@@ -0,0 +1,64 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+const { withHelpers } = require('@jungvonmatt/contentful-migrations');
+
+/**
+ * Contentful migration
+ * API: https://github.com/contentful/contentful-migration
+ * Editor Interfaces: https://www.contentful.com/developers/docs/extensibility/app-framework/editor-interfaces/
+ */
+
+const RESOURCE_CENTER_PAGE = 'pagePageResourceCenter';
+const LEGACY_COMPOSE_PAGE = 'page';
+module.exports = withHelpers(async function (migration, context, helpers) {
+    const pagePageResourceCenterMigration =
+        migration.editContentType(RESOURCE_CENTER_PAGE);
+
+    // 1. Add fields to our custom page that were previously on the Compose: Page model
+    pagePageResourceCenterMigration
+        .createField('title')
+        .name('Page title')
+        .type('Symbol')
+        .localized(true)
+        .required(false) // change this to True later on, after population
+        .validations([])
+        .disabled(false)
+        .omitted(false);
+    pagePageResourceCenterMigration
+        .createField('slug')
+        .name('Slug')
+        .type('Symbol')
+        .localized(true)
+        .required(false) // change this to True later on, after population
+        .validations([])
+        .disabled(false)
+        .omitted(false);
+    pagePageResourceCenterMigration
+        .createField('seo')
+        .name('SEO metadata')
+        .type('Link')
+        .linkType('Entry')
+        .localized(true)
+        .required(false)
+        .validations([])
+        .disabled(false)
+        .omitted(false);
+
+    // 2. Add the “Aggregate Root” annotation to the page type so it can work
+    // as a page without needing Compose:Page to parent it
+    pagePageResourceCenterMigration.setAnnotations([
+        'Contentful:AggregateRoot'
+    ]);
+
+    // 3. Activate the page type
```

Comment:
**nit (non-blocking)**: since this appears to be working without the published put request, maybe we can remove or update this comment?

Reply:
We could, yes, though I kept it in as a footnote of sorts. Easy either way